tests/unit/http/server_test.ts

```diff
@@ -369,6 +369,64 @@ Rhum.testPlan("http/server_test.ts", () => {
     );
   });
 
+  Rhum.testSuite("handleHttpRequestForVirtualPathAsset", () => {
+    Rhum.testCase(
+      "Should send /data/static_file.txt to /tests/data/sample_1.txt",
+      async () => {
+        let request = members.mockRequest(
+          "/data/static_file.txt",
+          "get",
+        );
+        const server = new Drash.Http.Server({
+          directory: ".",
```

Comment:
I mentioned above about you using `./` for app_3001 servers, but you're still using `.` here so i assume it doesn't break, so curious why the `/` is needed? Feel free to resolve all the threads about `directory: "./"` if your comment address what i just asked

Reply:
i changed the directory configs to be `.`